src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -226,6 +226,11 @@ class MFEMSidreDataCollection : public mfem::DataCollection
   /** If no mesh was associated with the collection, this method should be
       called before using any of the Load() methods to read parallel data. */
   void SetComm(MPI_Comm comm);
+
+  /// Set number of files for parallel writing
+  /** Allows the user to consolidate data for N ranks into M files, 0 < M <= N
```

Comment:
Please add that when `num_files` is less than or equal to 0, there will be one rank per file. 
And that setting `num_files` should be less than or equal to `N` (where `N` is the number of ranks)

Reply:
Added info to comment